src/android/java/io/jxcore/node/BtToRequestSocket.java

```diff
@@ -67,75 +63,64 @@ public BtToRequestSocket(BluetoothSocket socket, BtSocketDisconnectedCallBack ha
         mmOutStream = tmpOut;
     }
 
-    public void setPort(int port) {
-        mHTTPPort = port;
-    }
-
     public void run() {
+
         try {
             print_debug("start Socket with port: " + mHTTPPort);
+            mHTTPPort = 0; // just to make sure its zero
```

Comment:
removed the variable.

I would assume that the Java-Thread itself, can not be started twice: http://stackoverflow.com/questions/1215548/is-it-legal-to-call-the-start-method-twice-on-the-same-thread